sdk/samples/iot/paho_iot_hub_pnp_component_sample.c

```diff
@@ -306,15 +306,15 @@ static void create_and_configure_mqtt_client(void)
   iot_sample_create_mqtt_endpoint(
       SAMPLE_TYPE, &env_vars, mqtt_endpoint_buffer, sizeof(mqtt_endpoint_buffer));
 
-  // Initialize the hub client with the connection options.
+  // Initialize the pnp client with the connection options.
   az_iot_pnp_client_options options = az_iot_pnp_client_options_default();
   options.component_names = pnp_device_components;
   options.component_names_length = pnp_components_length;
   rc = az_iot_pnp_client_init(
       &pnp_client, env_vars.hub_hostname, env_vars.hub_device_id, model_id, &options);
   if (az_result_failed(rc))
   {
-    IOT_SAMPLE_LOG_ERROR("Failed to initialize hub client: az_result return code 0x%08x.", rc);
+    IOT_SAMPLE_LOG_ERROR("Failed to initialize pnp client: az_result return code 0x%08x.", rc);
```

Comment:
```suggestion
    IOT_SAMPLE_LOG_ERROR("Failed to initialize Plug and Play client: az_result return code 0x%08x.", rc);
```

Reply:
ehhhh see this is where it gets fuzzy. Since technically, the client, as specified in the API, is `pnp`